pkg/action/rollback.go

```diff
@@ -211,7 +211,7 @@ func (r *Rollback) performRollback(currentRelease, targetRelease *release.Releas
 	}
 
 	deployed, err := r.cfg.Releases.DeployedAll(currentRelease.Name)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "has no deployed releases") {
```

Comment:
What is this change for? It seems unrelated to the rest of this PR.

Reply:
It was actually related to a bug I found in the memory driver when I added the two test cases (see my change to memory.go). It was behaving differently and returning nil instead of throwing the "no deployed releases". When I corrected that I discovered another defect in the rollback where it was attempting to set previous releases as superceded but when there were no previous releases it was throwing an error instead of ignoring.  If you review the comments above, I originally had added a nil check in the upgrade.go but decided to fix the defect in the memory driver which made this other issue visible.

Reply:
Alright. That makes sense.